docs/serving/installing-cert-manager.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Installing cert-manager for TLS certificates"
+linkTitle: "Installing cert-manager"
+weight: 62
+type: "docs"
+---
+
+Install the  [Cert-Manager](https://github.com/jetstack/cert-manager) tool to obtain TLS 
+certificates that you can use for secure HTTPS connections in Knative. For more information 
+about enabling HTTPS connections in Knative, see 
+[Configuring HTTPS with TLS certificates](./using-an-ssl-cert.md).
+
+You can use cert-manager to either manually obtain certificates, or to enable Knative for 
+automatic certificate provisioning. If you want to enable Knative to automatically obtain
+new TLS certificates and renew existing ones, you must also configure the `config-certmanager` 
+ConfigMap. Complete instructions about automatic certificate provisioning are provided in 
```

Comment:
I would delete the part ` If you want to enable Knative to automatically obtain
new TLS certificates and renew existing ones, you must also configure the `config-certmanager` 
ConfigMap.` because:
1. It is a bit misleading as cert-manager could not renew the certificates that are obtained from other resources. cert-manager could only renew the certs obtained from it.
2. configuring the `config-certmanager` is included in the complete instructions. So we may not need to call it out. 
WDYT?

Reply:
SGTM
Thanks!